libgraph/include/katana/GraphTopology.h

```diff
@@ -154,11 +159,11 @@ class KATANA_EXPORT GraphTopology : public GraphTopologyTypes {
   ///@returns Degree of node N
   size_t degree(Node node) const noexcept { return edges(node).size(); }
 
-  PropertyIndex edge_property_index(const Edge& eid) const noexcept {
+  virtual PropertyIndex edge_property_index(const Edge& eid) const noexcept {
```

Comment:
I think this may be undesirable given that it can be in a critical path of our code

Reply:
I agree, but I don't see a way around it if we want to use the base topology pointer to point to a derived topology. Unfortunately, this is the only method that is overwritten in the derived classes, other methods are just added on top of what's already available in the base topology. I am open to suggestions.